### ISSUE TITLE
Error conect conect whith json coinmarketcap

### ISSUE THREAD

**Issue by take2k3** [mondeja/pymarketcap#29]
Hi Mondeja, thanks a lot for your work is very good. 

Recently i have a problem to list markets using pymarketcap, this is the code:

```
from pymarketcap import Pymarketcap
cmc = Pymarketcap(timeout=10)

cmc.exchanges() # Run Ok
cmc.markets("BTC", convert="BTC") # Fail
```

```
This the mensaje:
DEBUG: 
b''
b'https://files.coinmarketcap.com/generated/search/quick_search.json'
Traceback (most recent call last):
  File "C:\Users\....\probando_pymarket", line 8, in <module>
    cmc.markets("BTC", convert="BTC")
  File "pymarketcap\core.pyx", line 462, in pymarketcap.core.Pymarketcap.markets
  File "pymarketcap\core.pyx", line 478, in pymarketcap.core.Pymarketcap.markets
  File "pymarketcap\core.pyx", line 90, in pymarketcap.core.Pymarketcap.correspondences.__get__
  File "pymarketcap\core.pyx", line 142, in pymarketcap.core.Pymarketcap._cache_symbols_ids
  File "pymarketcap\core.pyx", line 135, in pymarketcap.core.Pymarketcap._quick_search
  File "pymarketcap\core.pyx", line 295, in pymarketcap.core.Pymarketcap._get
pymarketcap.errors.CoinmarketcapHTTPError: Status code -> 403 | Url -> https://files.coinmarketcap.com/generated/search/quick_search.json
[Finished in 6.3s]
```
Know why this?

Thx in advance.

**Comment by mondeja** [comment]
What version you are using? `python3 -c "import pymarketcap;print(pymarketcap.__version__);"`

Install again, I think that this is fixed at #28: `pip3 install pymarketcap --upgrade`

**Comment by take2k3** [comment]
Thanks a lot Mondeja, i upgrade wrapper and run correctly.  The version now is 3.9.136.

Thx for your very good work!!
